code/modules/antagonists/heretic/knowledge/void_lore.dm

```diff
@@ -243,25 +273,36 @@
 /datum/heretic_knowledge/ultimate/void_final/proc/on_life(mob/living/source, seconds_per_tick, times_fired)
 	SIGNAL_HANDLER
 
-	for(var/mob/living/carbon/close_carbon in view(5, source))
+	var/list/effective_range = range(10, source)
+
+	for(var/mob/living/carbon/close_carbon in effective_range)
 		if(IS_HERETIC_OR_MONSTER(close_carbon))
+			close_carbon.apply_status_effect(/datum/status_effect/void_conduit)
 			continue
 		close_carbon.adjust_silence_up_to(2 SECONDS, 20 SECONDS)
+		close_carbon.apply_status_effect(/datum/status_effect/void_chill, 1)
+		close_carbon.adjust_eye_blur(rand(0 SECONDS, 2 SECONDS))
+		close_carbon.adjust_bodytemperature(-30 * TEMPERATURE_DAMAGE_COEFFICIENT)
+
+	for(var/obj/machinery/door/affected_door in effective_range)
+		affected_door.take_damage(rand(60, 80))
+	for(var/obj/structure/door_assembly/affected_assembly in effective_range)
+		affected_assembly.take_damage(rand(60, 80))
+	for(var/obj/structure/window/affected_window in effective_range)
+		affected_window.take_damage(rand(20, 40))
+	for(var/obj/structure/grille/affected_grille in effective_range)
+		affected_grille.take_damage(rand(20, 40))
+
+	for(var/turf/affected_turf in effective_range)
+		var/datum/gas_mixture/environment = affected_turf.return_air()
+		environment.temperature *= 0.9
```

Comment:
Rather than iterating over `effective_range`, which could be up to 400-500 atoms, we should just iterate over it once and then typecheck to see if it's something worth affecting
```dm
for(var/atom/thing in range(10, source))
    if(iscarbon())
        ...
    if(iswindow() || isgrille())
        ...
    if(isairlock() || isairlockassembly())
        ...
    if(isturf())
       ...
```